### PR TITLE
Added logo assets and refactored home page

### PR DESCRIPTION
**Pull request by langleynick**
Added the logo files
Implemented Logo into the Navbar and Footer
refactored some styles in the home page, Nav, and Footer
Added brand description and Contact link to horizontal scroll 
#4 #23

### CODE REVIEW

File: components/Navbar.tsx [hunk 1]
```diff
@@ -41,32 +42,42 @@ export function Navbar() {
...
-					navbar ? 'max-h-screen' : 'max-h-16'
+					navbar ? 'max-h-screen' : 'max-h-20'
 				} bg-dark-600/50 p-4 rounded-md max-w-5xl mx-auto backdrop-blur-lg transition-all duration-500 ease-in-out overflow-hidden`}
 			>
 				<div className='flex flex-row justify-between items-center'>
-					<div>Logo</div>
-					<div className='sm:hidden text-light-500 text-2xl h-6'>
+					<div className='flex flex-row items-center gap-x-2'>
+						<Link href='/'>
+							<Image
```
Comment (aSouchereau) [inline components/Navbar.tsx:51]: Not sure about using this logo, I think we should try to get a non-stacked version similar to [this](http://citycyclery.ca/wp-content/uploads/2019/02/cropped-city-cylery-logo-blue.png).

**Comment by aSouchereau** [comment]
Also, the brands section still has some issues on mobile, looks fine on devtools, but a real device has horizontal scrolling. I'm not sure what could be causing it, might be better to make this a separate issue

File: components/Navbar.tsx (hunk 1 above)
Comment (amyers0217) [reply components/Navbar.tsx:51]: I agree!

**Comment by dylanreedx** [comment]
> Also, the brands section still has some issues on mobile, looks fine on devtools, but a real device has horizontal scrolling. I'm not sure what could be causing it, might be better to make this a separate issue

If you can find the cause it would be great. We should also add `overflow-x-hidden` on the layout component to make sure nothing else has overflow scrolling 😅

File: components/Navbar.tsx [hunk 2]
```diff
@@ -41,32 +42,42 @@ export function Navbar() {
...
+							src='/city-cyclery-logo-white.svg'
+							alt='City Cyclery Logo'
+							width={60}
```
Comment (dylanreedx) [inline components/Navbar.tsx:54]: Maybe make this a bit smaller, I'm not quite kept up on what's happening about the assets haha

**Pull request merged**